metrics/firebase/firestore/rules/firestore.rules

```diff
@@ -46,8 +47,8 @@ service cloud.firestore {
   }
 
   /// Checks if a user access is authorized.
-  function isAccessAuthorized(database) {
-    return request.auth.uid != null && isEmailValid(database)
+  function isSignedIn(database) {
+    return request.auth.uid != null && isEmailValid(database);
```

Comment:
Could you please leave both methods for now so it will simplify the review? I suppose that we can use new methods only for the collections we are changing and old leave as is.